ykman-gui/py/yubikey.py

```diff
@@ -746,4 +754,3 @@ def init_with_logging(log_level, log_file=None):
 def init():
     global controller
     controller = Controller()
-    controller.refresh()
```

Comment:
Do we still need to remove this?

Reply:
Yes. If a SKY key is already inserted, prior to starting ykman, the error message will not popup. The reason is that we call refresh() in init(), plus we have a timer that calls refresh() every 500ms. So in the beginning, we have basically two calls to refresh(), and the first one, from init(), will die because of the other call (from the timer).

Reply:
Ok!